examples/models/llama/CMakeLists.txt

```diff
@@ -121,6 +123,13 @@ if(EXECUTORCH_BUILD_KERNELS_CUSTOM)
   list(APPEND link_libraries custom_ops)
 endif()
 
+if(EXECUTORCH_BUILD_TORCHAO)
+  set(TORCHAO_BUILD_EXECUTORCH_OPS ON)
+  add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../../../third-party/ao/torchao/experimental ${CMAKE_CURRENT_BINARY_DIR}/../../../third-party/ao/torchao/experimental)
```

Comment:
I think you can do `${EXECUTORCH_ROOT}/third-party/ao/torchao/experimental`

Reply:
But one is the source directory and the other the binary directory?

Reply:
Yeah, I meant for the first one we can make it simpler. But it's up to you